ui/typedoc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "entryPointStrategy": "packages",
+  "entryPoints": [
+    "."
+  ],
```

Comment:
This tells it to treat this project like a monorepo and look at the workspaces in the `package.json` in the current directory. That means we don't have to explicitly list all the packages in here. (I'd annotate this in the file, but JSON doesn't support comments).